cvmfs/loader.cc

```diff
@@ -403,6 +406,7 @@ static fuse_args *ParseCmdLine(int argc, char *argv[]) {
   grab_mountpoint_ = cvmfs_options.grab_mountpoint;
   suid_mode_ = cvmfs_options.cvmfs_suid;
   disable_watchdog_ = cvmfs_options.disable_watchdog;
+  simple_options_parsing_ = cvmfs_options.simple_options_parsing;
```

Comment:
Older versions of the loader won't have this information.  Please add the logic to prepare for that (see comment below regarding versioning).